jwst/straylight/straylight.py

```diff
@@ -257,11 +257,18 @@ def clean_showers(input_model, allregions, shower_plane=3, shower_x_stddev=18.0,
     shower_high_reject : float, optional
         High percentile of pixels to reject
 
+    save_shower_model : bool
+        If set, a shower model is created and returned along with the cleaned input_model
+        array. If not, the `shower_model` returned is None
+
     Returns
     -------
     output : `~jwst.datamodels.IFUImageModel`
         Straylight-subtracted science data.
 
+    output_shower_model : `~jwst.datamodel.JwstDataModel` or None
```

Comment:
We should be specific about the datamodel returned here - ImageModel as currently written, or IFUImageModel if we update it.
```suggestion
    output_shower_model : `~jwst.datamodel.ImageModel` or None
```

Reply:
This should still be changed, to IFUImageModel.